.github/workflows/deploy-to-gcs.yml

```diff
@@ -49,5 +49,6 @@ jobs:
               repository/ $BUCKET
           gcloud storage cp --cache-control=no-store repository/timestamp.json $BUCKET
 
-          # invalidate CDN cache
-          gcloud compute url-maps invalidate-cdn-cache $LOAD_BALANCER --path "/*" --async
+          # invalidate CDN cache. This can take a while. We don't use "--async"
+          # because we want CDN to be testable when workflow finishes
+          gcloud compute url-maps invalidate-cdn-cache $LOAD_BALANCER --path "/*"
```

Comment:
Can you double check this won't cause the workflow to timeout? I'm not sure where that is configured, but we have seen >20 minute workflows time out on public-good-instance for synchronous terraform updates

Reply:
* job timeout default (and max value) is 360 minutes:  https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes
* None of the usage limits for GitHub hosted runners seem relevant to us: https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#usage-limits

I don't see anything that should timeout (on the other hand, I'm sure GitHub has other non-public heuristics as well: I'm sure there are license terms that define appropriate use).